src/supporting-data/citations/adapters/citationsConverter.ts

```diff
@@ -24,10 +25,17 @@ export type Citation = {
   title?: string;
   // Will always be a string, could be like '2000', or 'DEC-2000'.
   publicationDate?: string;
+  institute?: string;
```

Comment:
Wondering if there's a way to identify the attributes we're not displaying at the moment...